components/webcompat_reporter/browser/fields.h

```diff
@@ -21,6 +21,8 @@ inline constexpr char kLanguageFarblingField[] = "languageFarblingEnabled";
 inline constexpr char kBraveVPNEnabledField[] = "braveVPNEnabled";
 inline constexpr char kChannelField[] = "channel";
 inline constexpr char kVersionField[] = "version";
+inline constexpr char kCookiePolicyField[] = "cookie_policy";
+inline constexpr char kBlockScriptsField[] = "block_scripts";
```

Comment:
I will add these fields on the server-side. Please remember to update https://github.com/brave/brave-browser/wiki/Web-compatibility-reports